src/Dfc.CourseDirectory.Core/DataStore/Sql/QueryHandlers/GetLatestUnpublishedVenueUploadForProviderHandler.cs

```diff
@@ -1,27 +1,29 @@
 ﻿using System.Data.SqlClient;
+using System.Linq;
 using System.Threading.Tasks;
 using Dapper;
 using Dfc.CourseDirectory.Core.DataStore.Sql.Models;
 using Dfc.CourseDirectory.Core.DataStore.Sql.Queries;
+using Dfc.CourseDirectory.Core.Models;
 
 namespace Dfc.CourseDirectory.Core.DataStore.Sql.QueryHandlers
 {
-    public class GetLatestVenueUploadForProviderWithStatusHandler :
-        ISqlQueryHandler<GetLatestVenueUploadForProviderWithStatus, VenueUpload>
+    public class GetLatestUnpublishedVenueUploadForProviderHandler :
```

Comment:
Previous query name wasn't completely accurate; it didn't return the latest venue upload for a given provider and status and the name suggests, it returned the latest venue upload for a given provider *if* it has one of the provided statuses. Changing the name and behaviour here to match.

Reply:
👍🏻